_docs-sources/pipelines/security/branch-protection.md

```diff
@@ -1,4 +1,4 @@
-# Branch Protection
+# Sandwhich Protection
```

Comment:
Was this intentional?

Reply:
lol i'd imagine not. Let's change it back?